### PR TITLE
Added Novevember and NationJS conferences to events data

### PR DESCRIPTION
**Pull request by christianhoward**
Added a few JS conferences happening in November and December to the list.

### CODE REVIEW

File: static/data/2017/11/events.json [hunk 1]
```diff
@@ -106,5 +106,14 @@
     "type"  : "hackathon",
     "website" : "https://www.eventbrite.com/e/48-hour-tech-launch-tickets-37738421635",
     "logo"  : "https://cdn.evbuc.com/eventlogos/1887522/1041361741-1.jpg" 
+  },
+  {
+    "name"  : "Nodevember 2017",
+    "start" : "27/11/2017",
+    "end"   : "28/11/2017",
+    "location"  : "Nashville, TN",
```
Comment (geekyd) [inline static/data/2017/11/events.json]: Please add a country

File: static/data/2017/11/events.json [hunk 2]
```diff
@@ -106,5 +106,14 @@
...
+    "type"  : "conference",
+    "website" : "http://nodevember.org/",
+    "logo"  : ""
```
Comment (geekyd) [inline static/data/2017/11/events.json]: We can use `http://nodevember.org/img/logo-dark.png` as the logo

**Comment by christianhoward** [comment]
I've made those updates, so it should be good to go.

**Comment by geekyd** [comment]
@christianhoward Thanks for the PR.

**Pull request merged**